browser_patches/winldd/PrintDeps.cpp

```diff
@@ -95,8 +95,9 @@ const DepsMap getDependencies(const HMODULE hMod)
 }
 
 int printDependencies(const char* library)
-{
-    HMODULE hMod = LoadLibraryEx(library, NULL, DONT_RESOLVE_DLL_REFERENCES);
+{
+    SetDllDirectoryA(".");
+    HMODULE hMod = LoadLibraryEx(library, NULL, DONT_RESOLVE_DLL_REFERENCES | LOAD_LIBRARY_SEARCH_USER_DIRS | LOAD_LIBRARY_SEARCH_SYSTEM32);

```

Comment:
Shouldn't we call `SetDefaultDllDirectories(LOAD_LIBRARY_SEARCH_USER_DIRS | LOAD_LIBRARY_SEARCH_SYSTEM32)` or even `SetDefaultDllDirectories(LOAD_LIBRARY_SEARCH_DEFAULT_DIRS)` ? I suspect with the latter you may not need `SetDllDirectoryA(".");`

Reply:
Will try that!

Reply:
It didn't work @yury-s. 
I think that `LOAD_LIBRARY_SEARCH_APPLICATION_DIR` won't work because the app is the PrintDeps.
`LOAD_LIBRARY_SEARCH_SYSTEM32` is not enough because we have dlls in the chromium folder.
The [docs](https://docs.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-setdefaultdlldirectories?WT.mc_id=DT-MVP-5003814) say that to make `LOAD_LIBRARY_SEARCH_USER_DIRS` you need to call `AddDllDirectory` or `SetDllDirectory`.

What do you think?

Reply:
Let's leave it as is then. The app dir for the deps checker is different indeed I missed that fact.